index.bs

```diff
@@ -1564,6 +1565,90 @@ invoked, must run these steps:
 
 </div>
 
+# Drag and Drop Integration # {#drag-and-drop}
+
+During a <em>drag-and-drop operation</em>, dragged file and
+directory items are associated with [=file entries=] and [=directory entries=] 
+respectively. Each [=/entry=] is stored as an element in the <l spec=html>{{DragEvent|DragEvent's}}</l>
```

Comment:
I would actually just leave out the entire sentence starting with "Each entry is stored...". We could duplicate the `Additionally, each directory item is represented in the drag data store item list as a File. If it is accessed via getAsFile() a zero-length File is returned.` sentence from entries-API, as that is still true here as well, but it's probably fine to just leave that out as well.

Reply:
I just removed it.